package.json

```diff
@@ -35,5 +35,6 @@
     "@rollup/plugin-node-resolve": "^15.2.3",
     "rollup": "^4.14.1",
     "rollup-plugin-terser": "^7.0.2"
-  }
+  },
+  "types": "index.d.ts"
```

Comment:
Would it be better to package the type files together into the dist directory?  There may be multiple entry files in the future.

Reply:
In theory, this type file should only be used for retrieval, even if it is not placed in dist, it should be automatically read by idea, or do you have any good changes?

Reply:
How about
```
  "types": "dist/index.d.ts",
  "exports": {
    ".": {
      "require": "./dist/index.cjs.js",
      "import": "./dist/index.esm.js",
      "types": "./types/index.d.ts"
    },
    "./file": {
      "require": "./dist/file.cjs.js",
    }
  },
```

Reply:
@iteyelmp What do you think?

Reply:
Looks okay